lib/goth/token.ex

```diff
@@ -126,14 +170,24 @@ defmodule Goth.Token do
 
   See [Storing and retrieving instance metadata](https://cloud.google.com/compute/docs/storing-retrieving-metadata)
   for more information on metadata server.
+
+
+  #### Using a custom HTTP Client:
+
+      iex> http_request = fn options ->
+      ...>   req = Req.build(options[:method], options[:url], options)
+      ...>   req = Req.put_default_steps(req)
+      ...>   Req.run(req)
+      ...> end
```

Comment:
the idea is it is gonna be simply: `http_client: &Req.request/1`. ;) Don't worry about it though, I'll update the docs on master.